src/main/kotlin/io/slama/events/shusher.kt

```diff
@@ -24,7 +24,7 @@ class Shusher(
 
     private val roles = mutableMapOf<String, String>()
     private val random = Random(System.currentTimeMillis())
-    private val config = getShusherConfig()
+    private val config = BotConfiguration.shusher
```

Comment:
I'm not sure but this a reference or a value copy? Because I think that if we reload the config, the shusher keeps doing with the config it already has.